client/src/components/LandingPage/AboutUs.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import {
+  Layout, Row, Col, Button, Menu, Image, Card,
+} from 'antd';
+
+function AboutUs() {
+  return (
+    <div className="aboutus-sec">
+      <Row>
+        <Col span={24}>
+
+          <div className="about-title">
+            <h3>
+              <strong>All-In-One</strong>
+              {' '}
```

Comment:
what is all in one and why you need me to merge this 'Lorem, ipsum' in main branch?